source/geometry/FloatPoint3D.ooc

```diff
@@ -29,7 +30,8 @@ FloatPoint3D: cover {
 		if (r != 0.0f)
 			r = (this z / r) clamp(-1.0f, 1.0f) acos()
 		r
-	} }
+	}}
```

Comment:
It does? I seem to recall implementing this rule it so it would allow this particular case, but I could be wrong. Either way, I think this looks better, don't you?